gmm_objects: [pringle, brown_table0]
gravity: [0, 0, -9.81]
plugins:
- {plugin_type: <class 'gebsyas.simulator_plugins.FakeGMMObjectPublisher'>, topic_prefix: ''}
- map_frame: map
  objects: [pringle]
  plugin_type: <class 'iai_bullet_sim.ros_plugins.TFPublisher'>
tick_rate: 50
world:
  constraints: []
  objects:
  - color: [0.28500000000000003, 0.8888069465339584, 0.95, 1.0]
    extents: [0.03, 0.03, 0.24]
    geom_type: cylinder
    height: 0.24
    initial_pose:
      position: [0.8119763135910034, -0.055259451270103455, 0.8634161353111267]
      rotation: [0, 0, 0, 1]
    mass: 0.5
    name: pringle
    radius: 0.03
    type: rigid_body
  - color: [1.2784999907016754, 1, 0, 1]
    extents: [1.53000009059906, 0.7600000500679016, 0.7400000095367432]
    geom_type: box
    height: 0.7400000095367432
    initial_pose:
      position: [0.9979207515716553, -0.09882456064224243, 0.3788298964500427]
      rotation: [0.0, -0.0, -0.7056991457939148, 0.7085115909576416]
    mass: 0
    name: brown_table0
    radius: 0.76500004529953
    type: rigid_body
